tst/standard/conglatt.tst

```diff
@@ -17,7 +17,7 @@ gap> SEMIGROUPS.StartTest();
 #T# Robustness against infinite semigroups
 gap> S := FreeSemigroup(2);;
 gap> congs := CongruencesOfSemigroup(S);
-Error, Semigroups: SEMIGROUPS.LatticeOfXCongruences: usage,
+Error, Semigroups: SEMIGROUPS.PrincipalXCongruencePoset: usage,
 first argument <S> must be a finite semigroup,
```

Comment:
I'm not too fond of these errors that report an internal function name. It would be better if this said `Error, Semigroups: CongruencesOfSemigroup: usage,` ...